media/transcoder/transcoder_test.go

```diff
@@ -210,18 +209,9 @@ func checkGCSFileExists(t *testing.T, bucketName string, fileName string) {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
 	defer cancel()
 
-	it := client.Bucket(bucketName).Objects(ctx, nil)
-	for {
-		attrs, err := it.Next()
-		if err == iterator.Done {
-			break
-		}
-		if err != nil {
-			t.Fatalf("Bucket(%q).Objects: %v", bucketName, err)
-		}
-		if fileName == attrs.Name {
-			return
-		}
+	objAttrs, err := client.Bucket(bucketName).Object(fileName).Attrs(ctx)
+	if err == nil && objAttrs != nil {
```

Comment:
Should check if the error is `storage.ErrObjectNotExist`, since that is the error if the object doesn't exist.

Reply:
Done

Reply:
Done